test/js-native-api/test_object/test_object.c

```diff
@@ -472,6 +472,26 @@ static napi_value TestGetProperty(napi_env env,
   return object;
 }
 
+static napi_value TestFreeze(napi_env env,
+                             napi_callback_info info) {
+  napi_value object;
+
+  NAPI_CALL(env, napi_create_object(env, &object));
```

Comment:
You're passing an object into this function from JS, but you're not using the passed-in object. Instead, you create a new object and freeze/seal that.

It may be better to retrieve the passed-in object and freezing/sealing that.

Reply:
whoops, thought i updated this part 🤦🏻‍♀️